Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -686,6 +687,7 @@ private void RenderNodes (HtmlTextWriter writer, WebTreeNodeCollection nodes, bo
         writer.AddStyleAttribute ("white-space", "nowrap");
```

Comment:
The wrap-option doesn't work in the new CSS. The idea is, if the node is too long, it should wrap. It's okay if the nowrap becomes a css class on the treeview's root and is then applied via css file instead of via markup.

Reply:
Please elaborate, when should it wrap?

Reply:
![image](https://user-images.githubusercontent.com/6972548/133231503-bd49275b-9183-4e4b-97b1-44b89cabc31c.png)
You set a width on the tree, and the treeview will wrap the individual node instead of showing a scrollbar. It will show a horizontal scrollbar, when the node depth exceeds the width.

Reply:
Please have a look, I think I got it right.